packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -87,7 +87,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   NSDictionary *initProps = updateInitialProps([self prepareInitialProps], fabricEnabled);
 
-  RCTAppSetupPrepareApp(application, enableTM, *_reactNativeConfig);
+  RCTAppSetupPrepareApp(application, enableTM);
```

Comment:
Oh nice! This will make my PR easier #42263 (no need to separate this logic to another function). Commenting here to get notified when it gets merged 👍🏻

Reply:
Great :D It'll get merged as soon as I sort out the PR that's stacked with this (failing internal CI in a tricky way).